tests/common/testbed.py

```diff
@@ -204,7 +204,7 @@ def write_line_break(self, data=None):
                       explicit_start=True, Dumper=IncIndentDumper)
 
     def get_testbed_type(self, topo_name):
-        pattern = re.compile(r'^(t0|t1|ptf|fullmesh|dualtor)')
+        pattern = re.compile(r'^(t0|t1|ptf|fullmesh|dualtor|t2)')
```

Comment:
can we restrict to VoQ based T2 device ?

Reply:
is there a non VoQ based T2 ? 
how to determine if this is VoQ based T2 or not in testbed.py ?